samples/can/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
About the commit message of the sample. Does the commit message mean that you have only tested this with stm32f072b_disco, or is it really only working in that specific board. I did not see anything specific for just that board. If the sample should work in other smt32 boards too, then you could say that "Tested on stm32f072b_disco" instead of current text.

Also there should be a readme file for the sample, see examples what to put there for example in networking samples.